ci-operator/step-registry/stackrox/install-helm/stackrox-install-helm-commands.sh

```diff
@@ -0,0 +1,178 @@
+    command: ["tail", "-f", "/dev/null"]
+    volumeMounts:
+    - mountPath: "/tmp/helm-templates"
+      name: helm-template-volume
+  volumes:
+    - name: helm-template-volume
+      emptyDir:
+        sizeLimit: 50Mi
+EOF
+
+  echo "Content of: roxctl-extract-pod.yaml"
+  cat "${SCRATCH}/roxctl-extract-pod.yaml"
+
+  retry oc apply --namespace "${TMP_CI_NAMESPACE}" --filename "${SCRATCH}/roxctl-extract-pod.yaml"
+  retry oc wait --namespace "${TMP_CI_NAMESPACE}" --for=condition=Ready --timeout=60s pod/roxctl-extract-pod
+
+  retry oc exec --namespace "${TMP_CI_NAMESPACE}" roxctl-extract-pod --container main -- roxctl helm output central-services --image-defaults opensource --output-dir /tmp/helm-templates/central-services
+  retry oc exec --namespace "${TMP_CI_NAMESPACE}" roxctl-extract-pod --container main -- roxctl helm output secured-cluster-services --image-defaults opensource --output-dir /tmp/helm-templates/secured-cluster-services
+
+  retry oc cp --namespace "${TMP_CI_NAMESPACE}" --container rh-ubi roxctl-extract-pod:/tmp/helm-templates/central-services "${SCRATCH}/central-services"
+  retry oc cp --namespace "${TMP_CI_NAMESPACE}" --container rh-ubi roxctl-extract-pod:/tmp/helm-templates/secured-cluster-services "${SCRATCH}/secured-cluster-services"
+
+  retry oc delete pod --namespace "${TMP_CI_NAMESPACE}" roxctl-extract-pod
+  retry oc delete project "${TMP_CI_NAMESPACE}"
+}
+
+function install_central_with_helm() {
+  /tmp/helm/linux-amd64/helm upgrade --install --namespace stackrox --create-namespace stackrox-central-services "${SCRATCH}/central-services" \
+    --version "${ACS_VERSION_TAG}" \
+    --set imagePullSecrets.allowNone=true \
+    --set central.persistence.none=true \
+    --set central.adminPassword.value="${ROX_PASSWORD}"
+}
+
+function get_init_bundle() {
+  echo ">>> Get init-bundle and save to local helm values file"
+  function init_bundle() {
+    oc -n stackrox exec deploy/central -- \
+      roxctl central init-bundles generate my-test-bundle \
+        --insecure-skip-tls-verify --password "${ROX_PASSWORD}" --output - > "${SCRATCH}/helm-init-bundle-values.yaml"
+  }
+  retry init_bundle
+}
+
+function install_secured_cluster_with_helm() {
+  /tmp/helm/linux-amd64/helm upgrade --install --namespace stackrox --create-namespace stackrox-secured-cluster-services "${SCRATCH}/secured-cluster-services" \
+  --values "${SCRATCH}/helm-init-bundle-values.yaml" \
+  --set clusterName=remote \
+  --set imagePullSecrets.allowNone=true
+}
+
+fetch_last_nigthly_tag
+prepare_helm_templates
+install_helm
+
+install_central_with_helm
+echo ">>> Wait for 'stackrox-central-services' deployments"
+wait_deploy central-db
+wait_deploy central
+
+get_init_bundle
+install_secured_cluster_with_helm
+echo ">>> Wait for 'stackrox-secured-cluster-services' deployments"
+wait_deploy scanner
+wait_deploy scanner-db
+wait_deploy sensor
+wait_deploy admission-control
+
+retry oc get pods --namespace stackrox
```

Comment:
Check if `sensor` is connected to `central`. Ideally wait for the secured cluster to be ready.